src/helpers/validators.js

```diff
@@ -0,0 +1,13 @@
+import _ from 'lodash';
+
+export const validateNumber = input => {
+  return _.isNumber(input / 1);
```

Comment:
I am wondering on two things.

is `"1"` a number ? imo, it's a string ;) You are casting to number to check if it is a number... :P  if you still want to have such a behaviour, this name is super missleading.

Also, it is more modern approach instead of `input / 1` -> `Number(input)`